doc/code.html

```diff
@@ -384,7 +387,7 @@ <h3 id="Library">Your first library</h3>
 import (
 	"fmt"
 
-	<b>"github.com/user/stringutil"</b>
+	<b>"github.com/user/hello/stringutil"</b>
```

Comment:
Then we can revert this to not have the `hello` package
```suggestion
	<b>"github.com/user/stringutil"</b>
```